packages/api/src/models/animalUseModel.js

```diff
@@ -57,7 +57,7 @@ class AnimalUse extends baseModel {
     }, {});
 
     const response = Object.entries(usesPerType).map(([defaultTypeId, uses]) => {
-      return { default_type_id: defaultTypeId, uses };
+      return { default_type_id: +defaultTypeId, uses };
```

Comment:
This does fix the typing, but I am not sure about all the manipulation to format the data before it gets to the frontend that seems to be the root issue. 

I might look deeper tomorrow but this is a strange endpoint...